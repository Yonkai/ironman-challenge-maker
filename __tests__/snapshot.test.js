// example snapshot
describe(`<AAA/>`, () => {
  it('renders and matches the snapshot', () => {
    const price = '$220,000'
    expect(price).toMatchSnapshot()
  })
})
