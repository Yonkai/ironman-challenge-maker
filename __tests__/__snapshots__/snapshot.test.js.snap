// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AAA/> renders and matches the snapshot 1`] = `"$220,000"`;
